### PR TITLE
[ALLUXIO-2016] Remove @throws javadoc from unit tests

### PR DESCRIPTION
**Pull request by aaudiber**
https://alluxio.atlassian.net/browse/ALLUXIO-2016

### CODE REVIEW

File: core/client/src/test/java/alluxio/client/block/DelegatedUnderStoreBlockInStreamTest.java [hunk 1]
```diff
@@ -138,8 +134,7 @@ private void singleByteReadInternal(UnderStoreBlockInStream inStream, int startI
 
   /**
    * Tests that array read methods read the correct data, for the first block of the file.
-   *
-   * @throws IOException when reading from the stream fails
+
```
Comment (jsimsa) [inline core/client/src/test/java/alluxio/client/block/DelegatedUnderStoreBlockInStreamTest.java]: remove this line

File: core/server/src/test/java/alluxio/worker/block/evictor/LRFUEvictorTest.java [hunk 2]
```diff
@@ -96,7 +94,7 @@ private double calculateAccessWeight(long timeInterval) {
 
   /**
    * Sort all blocks in ascending order of CRF.
-   *
+
```
Comment (jsimsa) [inline core/server/src/test/java/alluxio/worker/block/evictor/LRFUEvictorTest.java]: add missing `*`

**Comment by jsimsa** [comment]
@aaudiber thanks, left some minor comments

**Comment by uronce-cc** [comment]
@aaudiber Ticket of 2016! Apart from comments by jiri, LGTM.

File: core/client/src/test/java/alluxio/client/block/DelegatedUnderStoreBlockInStreamTest.java (hunk 1 above)
Comment (aaudiber) [inline core/client/src/test/java/alluxio/client/block/DelegatedUnderStoreBlockInStreamTest.java]: You are a hawk, thanks for catching this!

File: core/server/src/test/java/alluxio/worker/block/evictor/LRFUEvictorTest.java (hunk 2 above)
Comment (aaudiber) [inline core/server/src/test/java/alluxio/worker/block/evictor/LRFUEvictorTest.java]: Fixed, thanks

**Comment by jsimsa** [comment]
LGTM

**Pull request merged**